### ISSUE TITLE
EOF error occurring on submission notification mailer

### ISSUE THREAD

**Issue by zazaian** [UM-USElab/gradecraft-development#1281]
EOF error happening on the following line in the submissions controller:

```ruby
85: NotificationMailer.successful_submission(@submission.id).deliver
```

Detailed further in this Rollbar issue:
https://rollbar.com/gradecraft/gradecraft-development/items/1304/

![image](https://cloud.githubusercontent.com/assets/95957/10851483/9f6302be-7f2b-11e5-8a14-c9cda8a87983.png)

**Comment by jwright** [comment]
This seems to be an issue with the [excon](https://github.com/excon/excon) gem that fog is using (http://stackoverflow.com/questions/19030306/why-do-i-get-intermittent-exconerrorssocketerror-eoferror-eoferror-errors) or it could be an issue with the SMTP server (http://stackoverflow.com/questions/3038852/ruby-mailer-is-coming-up-with-an-eoferror).

Can we ensure that we are using v 0.45 of excon in production? This issue was fixed in version 0.44 and our current Gemfile has excon pinned at 0.45.

I could not find any issue with Mandrill specifically for this issue

**Comment by chcholman** [comment]
@cshekhar can you check this out on Monday?
